SpatialGDK/Source/SpatialGDKServices/Private/LocalReceptionistProxyServerManager.cpp

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "LocalReceptionistProxyServerManager.h"
+
+#include "Internationalization/Regex.h"
+#include "Misc/FileHelper.h"
+#include "Serialization/JsonReader.h"
+#include "Serialization/JsonSerializer.h"
+#include "Serialization/JsonWriter.h"
+#include "Sockets.h"
+#include "SocketSubsystem.h"
+#include "UObject/CoreNet.h"
+
+#include "SpatialCommandUtils.h"
+#include "SpatialGDKServicesConstants.h"
+#include "..\Public\LocalReceptionistProxyServerManager.h"
+
+DEFINE_LOG_CATEGORY(LogLocalReceptionistProxyServerManager);
+
+#define LOCTEXT_NAMESPACE "FLocalReceptionistProxyServerManager"
+
+namespace
+{
+	static const FString ProxyInfoFilePath = FPaths::Combine(SpatialGDKServicesConstants::ProxyFileDirectory, SpatialGDKServicesConstants::ProxyInfoFilename);
```

Comment:
I'm leaning towards saving this in the SpatialGDKServicesConstants as well. Looking at where we use e.g. `ProxyInfoFilename` it's only ever in logs. And in logs I would prefer us to print the whole path so that people know where to look for it